src/modelAnalysis/topology/pathVectors.m

```diff
@@ -0,0 +1,195 @@
+function [output] = pathVectors(model, directory, varargin)
+% Computes elementary mode and extreme pathway
```

Comment:
The signature has to be changed to use optional arguments and not parameter/value pairs as required by @rmtfleming in #497